apps/bottle/src/app/not-found.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import BasketImage from '@/assets/basket.webp';
+import { Header } from '@/components/header';
+import { AppBridgeMessageType, useAppBridge } from '@/features/app-bridge';
+import { Asset, Button, Paragraph, spacings } from '@bottlesteam/ui';
+import Image from 'next/image';
+import { errorImageContainer } from './layout.css';
+
+export default function DefaultErrorPage() {
+  const { send } = useAppBridge();
+
+  const closeWebView = () => send({ type: AppBridgeMessageType.WEB_VIEW_CLOSE });
```

Comment:
Is there a convention for separating this into a handler function? 
I'm asking because when I look at [this line](https://github.com/Nexters/Bottles_Web/pull/38/files#diff-7d1e4dd3c156dc45cbde80d76faef55591b21376eb4abbd9349fc35ee7b1d9f1R28), it seems to have a similar structure but doesn't create a separate handler function!

Reply:
I separated this just because prevent repeating. 
`send` is called twice with same arguments at line27, line 18